.github/workflows/ci.yml

```diff
@@ -42,14 +82,25 @@ jobs:
     # https://github.com/haskell/actions/issues/36
     # We also use --enable-tests and --enable-benchmarks to avoid
     # test and bench commands from reconfiguring and thus rebuilding.
-    - name: Test
+    - name: Build Test
       run: |
+        cabal user-config diff
```

Comment:
What is this for?

Reply:
A ghcup bootstrap script bug once broke windows runners. This helps with debugging.